kfs-core/src/main/resources/institutional-config.properties

```diff
@@ -1,3 +1,6 @@
+batchinvoker.directory.towatch=${work.directory}/control
+batchinvoker.directory.history=${work.directory}/control/history
```

Comment:
How are these config properties used? It looks like these values are set in the Spring file.